modules/ROOT/pages/clauses/match.adoc

```diff
@@ -262,151 +228,262 @@ RETURN a, b
 
-For example, the following query creates a relationship which contains a space (`OLD FRIENDS`) between `Martin Sheen` and `Rob Reiner`.
+[[find-paths]]
+== Find paths
 
-.Query
-[source, cypher, indent=0]
+The `MATCH` clause can also be used to bind whole paths to variables.
+
+.Find all paths matching a pattern
+[source, cypher]
 ----
-MATCH
-  (martin:Person {name: 'Martin Sheen'}),
-  (rob:Person {name: 'Rob Reiner'})
-CREATE (rob)-[:`OLD FRIENDS`]->(martin)
+MATCH path = ()-[:ACTED_IN]->(movie:Movie)
+RETURN path
 ----
 
-This leads to the following graph:
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| path
+| (:Person {name: "Charlie Sheen"})-[:ACTED_IN {role: "Bud Fox"}]->(:Movie {title: "Wall Street"}) 
+| (:Person {name: "Martin Sheen"})-[:ACTED_IN {role: "Carl Fox"}]->(:Movie {title: "Wall Street"})
+| (:Person {name: "Martin Sheen"})-[:ACTED_IN {role: "A.J. MacInerney"}]->(:Movie {title: "The American President"})
+| (:Person {name: "Michael Douglas"})-[:ACTED_IN {role: "Gordon Gekko"}]->(:Movie {title: "Wall Street"})
+| (:Person {name: "Michael Douglas"})-[:ACTED_IN {role: "President Andrew Shepherd"}]->(:Movie {title: "The American President"})
+| Rows: 5
+|===
 
-image::graph_match_clause_backtick.svg[width="600", role="middle"]
 
-.Query
-[source, cypher, indent=0]
+.Find paths matching a pattern including a `WHERE` predicate
+[source, cypher]
 ----
-MATCH (n {name: 'Rob Reiner'})-[r:`OLD FRIENDS`]->()
-RETURN type(r)
+MATCH path = (:Person)-[:ACTED_IN]->(movie:Movie)<-[:DIRECTED]-(:Person)
+WHERE movie.title = 'Wall Street'
+RETURN path
 ----
 
 .Result
 [role="queryresult",options="header,footer",cols="1*<m"]
 |===
-| type(r)
-| "OLD FRIENDS"
-|Rows: 1
+| path
+| (:Person {name: "Charlie Sheen"})-[:ACTED_IN {role: "Bud Fox"}]->(:Movie {title: "Wall Street"})<-[:DIRECTED]-(:Person {name: "Oliver Stone"})
+| (:Person {name: "Martin Sheen"})-[:ACTED_IN {role: "Carl Fox"}]->(:Movie {title: "Wall Street"})<-[:DIRECTED]-(:Person {name: "Oliver Stone"})
+| (:Person {name: "Michael Douglas"})-[:ACTED_IN {role: "Gordon Gekko"}]->(:Movie {title: "Wall Street"})<-[:DIRECTED]-(:Person {name: "Oliver Stone"})
+|Rows: 3
 |===
 
+For more information about how `MATCH` is used to find patterns of varying complexity (including xref:patterns/variable-length-patterns.adoc#quantified-path-patterns[quantified path patterns], xref:patterns/variable-length-patterns.adoc#quantified-relationships[quantified relationships], and the xref:patterns/shortest-paths.adoc[shortest paths] between nodes), see the section on xref::patterns/index.adoc[Patterns].
 
-[[multiple-rels]]
-=== Multiple relationships
+== Multiple MATCH clauses, the WITH clause, and clause composition
 
-Relationships can be expressed by using multiple statements in the form of `()--()`, or they can be strung together.
-For example:
+In Cypher, a query’s behavior is defined by its clauses.
```

Comment:
```suggestion
In Cypher, a query's behavior is defined by its clauses.
```
alternatively, of-construction instead of genitive s
(suggestion was about the tick character though. there might be more occurrences of this character)